files/dhcp/dhclient.conf

```diff
@@ -19,7 +19,7 @@ option acl-url code 226 = text;
 send host-name = gethostname();
 request subnet-mask, broadcast-address, time-offset, routers,
         domain-name, domain-name-servers, domain-search, host-name,
-        dhcp6.name-servers, dhcp6.domain-search,
-        netbios-name-servers, netbios-scope, interface-mtu,
-        rfc3442-classless-static-routes, ntp-servers, snmp-community, minigraph-url;
+        dhcp6.name-servers, dhcp6.domain-search, interface-mtu,
```

Comment:
No I didn't. I'm not exactly sure which of these dhclient option names maps to option 119.